Core/src/org/sleuthkit/autopsy/modules/interestingitems/FilesSetRulePanel.java

```diff
@@ -135,12 +135,18 @@ private void populatePathFilterComponents(FilesSet.Rule rule) {
      */
     boolean isValidRuleDefinition() {
         // The rule must have a name.
+        // If rule name is absent, set it as the Name Pattern since nameTextField cannot be left empty.
         if (this.ruleNameTextField.getText().isEmpty()) {
-            NotifyDescriptor notifyDesc = new NotifyDescriptor.Message(
-                    NbBundle.getMessage(FilesSetPanel.class, "FilesSetRulePanel.messages.filesSetRulesMustBeNamed"),
-                    NotifyDescriptor.WARNING_MESSAGE);
-            DialogDisplayer.getDefault().notify(notifyDesc);
-            return false;
+            String ruleName = this.nameTextField.getText();
+            if (ruleName != null && !ruleName.isEmpty()) {
```

Comment:
it is probably not worth changing this pull request But i have been using StringUtils.notBlank() with a static import instead of the `ruleName  != null && !ruleName.isEmpty()`. I think it is more concise and readable.

in this example line 141 would become:
`if (notBlank(ruleName){`